packages/open-next/src/overrides/wrappers/cloudflare-node.ts

```diff
@@ -52,17 +49,15 @@ const handler: WrapperHandler<InternalEvent, InternalResult> =
           status: statusCode,
           headers: responseHeaders,
         });
-        resolveResponse(response);
 
         return Writable.fromWeb(writable);
       },
       onWrite: () => {},
       onFinish: (_length: number) => {},
     };
 
-    ctx.waitUntil(handler(internalEvent, streamCreator));
-
-    return promiseResponse;
+    const internalResult = await handler(internalEvent, streamCreator);
```

Comment:
It may fix it, but it also means that streaming will not work.
The handler in node returns only after the response is fully sent (and the streamCreator is not useful in this case)

Maybe using a custom `Writable` with a custom `ReadableStream` would help figure out what's the issue here (or resolving the response in `onWrite` (this may break other things, but at least it'll help figure out what's going on here)